docs/guides/real-time/getting-started/websockets.md

```diff
@@ -202,68 +194,62 @@ operations over the connection. You have also created your first subscription.
 <html>
 	<body>
 		<script>
-			const url = 'wss://your-directus-url/websocket';
-			const access_token = 'your-access-token';
-			const collection = 'messages';
 
-			const connection = new WebSocket(url);
-
-			connection.addEventListener('open', function () {
-				console.log({ event: 'onopen' });
-				connection.send(
-					JSON.stringify({
-						type: 'auth',
-						access_token,
-					})
-				);
-			});
-
-			connection.addEventListener('message', function (message) {
-				const data = JSON.parse(message.data);
-				console.log({ event: 'onmessage', data });
-			});
-
-			connection.addEventListener('close', function () {
-				console.log({ event: 'onclose' });
-			});
-
-			connection.addEventListener('error', function (error) {
-				console.log({ event: 'onerror', error });
-			});
-
-			function subscribe() {
-				connection.send(
-					JSON.stringify({
-						type: 'subscribe',
-						collection: 'messages',
-						query: {
-							fields: ['*'],
-						},
-					})
-				);
-			}
-
-			function createItem(text, user) {
-				connection.send(
-					JSON.stringify({
-						type: 'items',
-						collection: 'messages',
-						action: 'create',
-						data: { text, user },
-					})
-				);
-			}
-
-			function readLatestItem() {
-				connection.send(
-					JSON.stringify({
-						type: 'items',
-						collection: 'messages',
-						action: 'read',
-						query: { limit: 1, sort: '-date_created' },
-					})
-				);
-			}
+      import { createDirectus, staticToken, realtime } from "https://www.unpkg.com/@directus/sdk@11.0.3/dist/index.js"
+
+      const url = 'wss://your-directus-url/websocket';
+      const access_token = 'your-access-token';
+      const collection = 'messages';
+
+      const client = createDirectus(url)
+        .with(staticToken(access_token))
+        .with(realtime());
+
+      await client.connect();
```

Comment:
This is not wrong but also not technically required 😄 `client.subscribe` will also attempt to connect to the websocket if a connection has not yet been established

Reply:
explicitly opening the connection seems a lot less error prone than relying on the `subscribe` function to do this so marking this as resolved